console/src/components/Common/TableCommon/DragFoldTable.js

```diff
@@ -3,7 +3,10 @@
 import React, { Component } from 'react';
 import ReactTable, { ReactTableDefaults } from 'react-table';
 import 'react-table/react-table.css';
+
 import FoldableHoc from './foldableTable';
+import Spinner from '../Spinner/Spinner';
+import styles from '../Common.scss';
 
 Object.assign(ReactTableDefaults, {
```

Comment:
Sorry I missed this in the `browse rows improvements` PR, but can you remove these defaults and pass them to the ReactTable component directly where it's used?

Just to make the code declarative. There's no way anybody will read these defaults if they are reading the component code. Do you agree?

Reply:
I removed it because:
1) defaultPageSize is already 10,
2) minRows wasn't enforced in react-table anyway.